modules/host/host.go

```diff
@@ -105,8 +106,19 @@ func New(cs *consensus.ConsensusSet, hdb modules.HostDB, tpool modules.Transacti
 	if err != nil {
 		return nil, err
 	}
+	host, err := modules.IGD.ExternalIP()
+	if err != nil {
+		// log error? should be fatal
```

Comment:
I don't think it should be fatal. We want people to include the host module by default, even if they are having trouble finding their external IP address (we don't want siad to exit just because the host can't be used).

Logging an error is probably okay.

Does IGD.ExternalIP work if upnp is not supported?